module/data/item/feat.mjs

```diff
@@ -179,7 +179,7 @@ export default class FeatData extends ItemDataModel.mixin(
    * @param {object} source  The candidate source data from which the model will be constructed.
    */
   static #migrateEnchantment(source) {
-    if ( !("enchantment" in source) || !("items" in source.enchantment) ) return;
+    if ( foundry.utils.getType(source.enchantment) !== "Object" ) return;
```

Comment:
Doesn't this throw an Error now for partial updates to `enchantment` that do not include the `items` property?

Reply:
Good point (had to change it because the old one was throwing an error when `source.enchantment` was `null`)